metricbeat/module/couchdb/server/_meta/data.json

```diff
@@ -0,0 +1,58 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
```

Comment:
As the indentation is off in this file and contains some old fields, I wonder if this was generated or manually created?

Reply:
This was generated. Should I update this file?

Reply:
Yes, can you generated it again to see if the strange indentation disappears?